nilearn/datasets/struct.py

```diff
@@ -169,12 +197,52 @@ def load_mni152_brain_mask():
     return mask_img
 
 
+def load_mni152_gm_mask():
+    """Load brain mask from gray-matter MNI152 template
+
+    Returns
+    -------
+    mask_img : Nifti-like mask image corresponding to gray matter.
+
+    """
+    # Load MNI template
+    gm_target = load_mni152_gm_template()
+    gm_target_img = check_niimg(gm_target)
+    gm_target_data = niimg._safe_get_data(gm_target_img)
+
+    gm_target_mask = (gm_target_data > 0.2)
```

Comment:
I'm not sure what we should do with this hard-coded threshold... :thinking: 
`load_mni152_brain_mask` uses 0 but if we set it at 0 here, the behavior isn't consistent with what `fetch_icbm152_brain_gm_mask` does. If we leave it at 0.2 it might be a source of confusion for users.
Any ideas @bthirion ?

Reply:
Use an optional `threshold` argument

Reply:
> In addition, we also need to decide whether we should update the brain template to have the same resolutions (see one of my previous comments).

Yes, we should. I guess the standard resolutions are 1mm and 2mm.